### ISSUE TITLE
Build process fails on Windows

### ISSUE THREAD

**Issue by crbinz** [JuliaGraphics/Tk.jl#134]
```
               _                                                                              
   _       _ _(_)_     |  A fresh approach to technical computing                             
  (_)     | (_) (_)    |  Documentation: http://docs.julialang.org                            
   _ _   _| |_  __ _   |  Type "?help" for help.                                              
  | | | | | | |/ _` |  |                                                                      
  | | |_| | | | (_| |  |  Version 0.5.1 (2017-03-05 13:25 UTC)                                
 _/ |\__'_|_|_|\__'_|  |  Official http://julialang.org/ release                              
|__/                   |  x86_64-w64-mingw32                                                  
                                                                                              
julia> Pkg.build("Tk")                                                                        
INFO: Building WinRPM                                                                         
INFO: Downloading https://cache.julialang.org/http://download.opensuse.org/repositories/window
s:/mingw:/win32/openSUSE_42.2/repodata/repomd.xml                                             
INFO: Downloading https://cache.julialang.org/http://download.opensuse.org/repositories/window
s:/mingw:/win32/openSUSE_42.2/repodata/8c314b3318fcfdf09df6ae574d339eab913a0271181296ecebec87e
b4233a5a8-primary.xml.gz                                                                      
INFO: Downloading https://cache.julialang.org/http://download.opensuse.org/repositories/window
s:/mingw:/win64/openSUSE_42.2/repodata/repomd.xml                                             
INFO: Building Cairo                                                                          
INFO: Updating WinRPM package list                                                            
INFO: Downloading https://cache.julialang.org/http://download.opensuse.org/repositories/window
s:/mingw:/win32/openSUSE_42.2/repodata/repomd.xml                                             
INFO: Downloading https://cache.julialang.org/http://download.opensuse.org/repositories/window
s:/mingw:/win64/openSUSE_42.2/repodata/repomd.xml                                             
INFO: Building Tk                                                                             
ERROR: Build process failed.                                                                  
 in build!(::Array{String,1}, ::Dict{Any,Any}, ::Set{Any}) at .\pkg\entry.jl:631              
 in build(::Array{String,1}) at .\pkg\entry.jl:650                                            
 in (::Base.Pkg.Dir.##2#3{Array{Any,1},Base.Pkg.Entry.#build,Tuple{Array{String,1}}})() at .\p
kg\dir.jl:31                                                                                  
 in cd(::Base.Pkg.Dir.##2#3{Array{Any,1},Base.Pkg.Entry.#build,Tuple{Array{String,1}}}, ::Stri
ng) at .\file.jl:48                                                                           
 in #cd#1(::Array{Any,1}, ::Function, ::Function, ::Array{String,1}, ::Vararg{Array{String,1},
N}) at .\pkg\dir.jl:31                                                                        
 in build(::String, ::Vararg{String,N}) at .\pkg\pkg.jl:236                                   
```

**Comment by Lemures** [comment]
Got this problem too.

**Comment by aviks** [comment]
Fixed by #147. Will be more reliable in #148
